File: httpsnippet/src/test/resources/output/javascript/jquery/full.js
var settings = {"async":true,"headers":{"Cookie":"foo=bar;bar=baz","content-type":"application/x-www-form-urlencoded","accept":"application/json"},"method":"POST","crossDomain":true,"body":{"foo":"bar"},"url":"http://mockbin.com/har?key=value"}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/test/resources/output/java/okhttp/full.java
OkHttpClient client = new OkHttpClient();

RequestBody body = new FormBody.Builder()
        .add("foo", "bar")
        .build();
Request request = new Request.Builder()
  .url("http://mockbin.com/har?baz=abc&foo=bar&foo=baz&key=value")
  .post(body)
  .addHeader("Cookie", "foo=bar; bar=baz")
  .addHeader("content-type", "application/x-www-form-urlencoded")
  .addHeader("accept", "application/json")
  .build();

Response response = client.newCall(request).execute();
